core/src/main/java/com/novoda/noplayer/subtitles/KitkatSubtitlesStyle.java

```diff
@@ -0,0 +1,59 @@
+package com.novoda.noplayer.subtitles;
+
+import android.graphics.Typeface;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+
+import com.google.android.exoplayer2.text.CaptionStyleCompat;
+
+import static android.os.Build.VERSION_CODES.KITKAT;
+
+@RequiresApi(api = KITKAT)
+class KitkatSubtitlesStyle implements SubtitlesStyle {
```

Comment:
Should be `KitKat` 😂 I'm being pedantic because KitKat is two words 👅

Reply:
Doing this change, however it depends if we spell it like `Kit Kat` or `KitKat`. If it's the second then it would follow the similar rules for acronyms, `HTTP -> Http`.

And the official website spells it as one word 🙃 

![image](https://user-images.githubusercontent.com/260697/65500689-931c5600-deb7-11e9-98a1-79402cd204b9.png)

I like being pedantic too 😅

Reply:
https://github.com/novoda/no-player/pull/277/commits/e94a6beadc23aad21c86cbdfbd4f1c3fb93708d4